examples_utils/notebook_logging/__init__.py

```diff
@@ -1,3 +1,12 @@
 # Copyright (c) 2023 Graphcore Ltd. All rights reserved.
 
+import subprocess
+
+from pathlib import Path
 from .gc_logger import *
+
+# Help IPython find our custom extension
+extension_path = Path(__file__).parent.joinpath("cell_logger.py").resolve()
+destination_path = Path("/root/.ipython/extensions").resolve()
+
+subprocess.run(f"cp {extension_path} {destination_path}", shell=True)
```

Comment:
Could use shutil copy?

Reply:
I'm sure I tried but for some reason paperspace/ubuntu/god really didnt like shutil copying a file to /root/.ipython/extensions

I can try again but this should be fine no? Under what conditions would shutil be better?

Reply:
This is fine